solidity/contracts/Coordinator.sol

```diff
@@ -160,6 +149,38 @@ contract Coordinator is ChainlinkRequestInterface, CoordinatorInterface {
     emit NewServiceAgreement(serviceAgreementID, _requestDigest);
   }
 
+  function fulfillData(
+    uint256 _requestId,
+    bytes32 _data
+  )
+    external
+    isValidResponse(_requestId)
+    returns (bool)
+  {
+    bytes32 requestId = bytes32(_requestId);
+    callbacks[requestId].responses[msg.sender] = uint256(_data);
+
+    Callback memory callback = callbacks[requestId];
+    callbacks[requestId].responseCount += 1;
+    uint256 responseCount = callbacks[requestId].responseCount;
+    bytes32 sAId = callbacks[requestId].sAId;
+    address[] memory oracles = serviceAgreements[sAId].oracles;
+    if (oracles.length > responseCount) {
+      return true;
+    }
+
+    uint256 result;
+    for (uint i = 0; i < responseCount; i++) {
+      result = result.add(callbacks[requestId].responses[oracles[i]]);
```

Comment:
I think we will still want to delete each response within the child mapping. I did a test [here](https://github.com/thodges-gh/SolidityStorageTests) that shows the responses won't be deleted when the callback is deleted. Should be able to save some gas.